public/app/features/browse-dashboards/components/BrowseActions/BrowseActions.tsx

```diff
@@ -69,15 +71,29 @@ export function BrowseActions({ folderDTO }: Props) {
   };
 
   const showMoveModal = () => {
-    appEvents.publish(
-      new ShowModalReactEvent({
-        component: MoveModal,
-        props: {
-          selectedItems,
-          onConfirm: onMove,
-        },
-      })
-    );
+    if (hasProvisioned && hasNonProvisioned && provisioningEnabled) {
```

Comment:
Have you updated the branch to the latest main? I think this was a part of the other PR?